src/Http/Controllers/CP/Collections/CollectionsController.php

```diff
@@ -24,7 +24,8 @@ public function index()
         $this->authorize('index', CollectionContract::class, __('You are not authorized to view collections.'));
 
         $collections = Collection::all()->filter(function ($collection) {
-            return User::current()->can('view', $collection);
+            return User::current()->can('view', $collection)
```

Comment:
If we modify this to check for 'configure collections' permission first, we can implement this improvement without affecting the current UX for those with elevated permissions.

```
return User::current()->can('configure collections')
    || User::current()->can('view', $collection)
    && $collection->sites()->contains(Site::selected()->handle());
```

Thoughts @duncanmcclean and @jasonvarga ?